src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -778,7 +778,9 @@ private function preBoot(): ContainerInterface
             $this->startTime = microtime(true);
         }
         if ($this->debug && !isset($_ENV['SHELL_VERBOSITY']) && !isset($_SERVER['SHELL_VERBOSITY'])) {
-            putenv('SHELL_VERBOSITY=3');
+            if (is_callable('putenv')) {
```

Comment:
You should use `\function_exists('putenv')` instead, as this will allow OPCache to optimise this to a `if (true)` (and then removing the `if` entirely) for the common case where this function is not disabled (this optimisation of OPCache works for functions defined in PHP core or its extensions, but this is where the function will be defined in this case for most users)